app/src/main/java/com/github/swent/echo/compose/components/TopBar.kt

```diff
@@ -1,273 +1,11 @@
-                        icon = {
-                            Icon(
-                                imageVector =
-                                    if (index == selectedItemIndex) {
-                                        item.selectedIcon
-                                    } else {
-                                        item.unselectedIcon
-                                    },
-                                contentDescription = item.title
-                            )
-                        },
-                        badge = {
-                            item.badgeCount?.let { Text(text = item.badgeCount.toString()) }
-                        },
-                        modifier =
-                            Modifier.padding(NavigationDrawerItemDefaults.ItemPadding)
-                                .testTag("navigation_item_$index")
-                    )
-                }
-            }
-        },
-        drawerState = drawerState,
-        modifier = Modifier.testTag("hamburger_menu")
-    ) {
-        // Top app bar with title, navigation icon, and actions
-        /* Implemented inside the content of the ModalNavigationDrawer because
-        it was not possible to use the hamburger menu independently of the topBar
-        , and saw a lot of examples that implemented it like this */
-        CenterAlignedTopAppBar(
-            colors =
-                TopAppBarDefaults.topAppBarColors(
-                    containerColor = MaterialTheme.colorScheme.primaryContainer,
-                    titleContentColor = MaterialTheme.colorScheme.onPrimary,
-                ),
-            title = {
-                Text(
-                    "Echo",
-                    maxLines = 1,
-                    overflow = TextOverflow.Ellipsis, // should not happen
-                    fontWeight = FontWeight.SemiBold
-                )
-            },
-            navigationIcon = { // hamburger menu
-                IconButton(
-                    onClick = { scope.launch { drawerState.open() } },
-                    modifier = Modifier.testTag("menu_button")
-                ) {
-                    Icon(
-                        imageVector = Icons.Filled.Menu,
-                        contentDescription = "Menu to access all subpages"
-                    )
-                }
-            },
-            actions = { // search icon
-                IconButton(
-                    onClick = onOpenSearch,
-                    modifier = Modifier.testTag("list_map_mode_button")
-                ) {
-                    Icon(
-                        imageVector = Icons.Filled.List,
-                        contentDescription = "Search icon to access the search screen"
-                    )
-                }
-            },
-            scrollBehavior = scrollBehavior,
-            modifier = Modifier.testTag("top_bar")
-        )
-    }
-}
+@Composable fun TopBar(navActions: NavigationActions? = null) {}
```

Comment:
It could also be best to remove this file. What do you think?

Reply:
also true